sql/synth_recipes.sql

```diff
@@ -984,50 +984,50 @@ INSERT INTO `synth_recipes` VALUES (13053,0,0,0,70,0,0,0,0,0,0,4096,4238,654,852
 INSERT INTO `synth_recipes` VALUES (13520,0,0,0,76,0,0,0,0,0,0,4096,4238,645,756,756,756,0,0,0,0,757,757,757,757,1,1,1,1,'Durium Ingot');
--- INSERT INTO `synth_recipes` VALUES (13521,0,0,0,76,0,0,255,0,0,0,4096,4238,648,653,653,653,657,715,818,852,16967,16989,16989,16989,1,1,1,1,'Mikazuki');
+INSERT INTO `synth_recipes` VALUES (13521,0,0,4,76,0,0,7,0,0,0,4096,4238,648,653,653,653,657,715,818,852,16967,16989,16989,16989,1,1,1,1,'Mikazuki'); -- 2 src agree, 2013 talk: 5/5 no skillup 19WW, 15LE
 INSERT INTO `synth_recipes` VALUES (13522,1,0,0,76,0,0,0,0,0,0,4100,4242,1118,0,0,0,0,0,0,0,649,654,654,654,2,1,1,1,'Antican Pauldron (desynth)');
 INSERT INTO `synth_recipes` VALUES (13523,0,0,0,76,0,0,0,0,0,0,4099,4241,9489,0,0,0,0,0,0,0,17784,17784,17784,17784,1,1,1,1,'Keppu');
 INSERT INTO `synth_recipes` VALUES (13524,0,0,0,77,0,0,0,0,0,0,4096,4238,654,654,654,851,0,0,0,0,16577,16828,16828,16828,1,1,1,1,'Bastard Sword');
--- INSERT INTO `synth_recipes` VALUES (13525,0,0,0,77,255,0,0,0,0,0,4096,4238,652,654,711,730,745,914,0,0,17222,18687,18687,18687,1,1,1,1,'Hexagun');
+INSERT INTO `synth_recipes` VALUES (13525,0,0,43,77,45,0,0,0,0,0,4096,4238,652,654,711,730,745,914,0,0,17222,18687,18687,18687,1,1,1,1,'Hexagun'); -- JPWiki/xiclopedia, others ??subs, no useful talk
 INSERT INTO `synth_recipes` VALUES (13526,0,0,0,77,0,0,0,0,0,0,4096,4238,757,0,0,0,0,0,0,0,758,758,758,758,1,1,1,1,'Durium Sheet');
 INSERT INTO `synth_recipes` VALUES (13527,0,0,0,77,0,0,0,0,0,0,4096,4238,652,654,654,0,0,0,0,0,17063,17435,17435,17435,1,1,1,1,'Darksteel Rod');
 INSERT INTO `synth_recipes` VALUES (13528,0,1995,0,77,0,0,0,0,0,0,4096,4238,757,757,757,757,757,757,2144,0,758,758,758,758,6,6,6,6,'Durium Sheet');
 INSERT INTO `synth_recipes` VALUES (13529,0,0,19,78,0,0,0,0,0,0,4096,4238,654,654,654,713,824,0,0,0,16789,16790,16790,16790,1,1,1,1,'Darksteel Scythe'); -- JPWiki recipe… most others only ??WW
 INSERT INTO `synth_recipes` VALUES (13530,1,0,0,78,0,0,0,0,0,0,4098,4240,1118,0,0,0,0,0,0,0,660,664,664,664,2,1,1,1,'Antican Pauldron (desynth)');
 INSERT INTO `synth_recipes` VALUES (13531,0,0,0,78,0,0,45,0,0,0,4099,4241,664,664,664,855,855,0,0,0,12580,13765,13765,13765,1,1,1,1,'Darksteel Harness'); -- FFXIAH post tested LE to be 45, not 43?
 INSERT INTO `synth_recipes` VALUES (13532,0,1993,0,78,0,0,0,0,0,0,4096,4238,2230,2233,18207,0,0,0,0,0,18493,18493,18493,18493,1,1,1,1,'Regiment Kheten');
--- INSERT INTO `synth_recipes` VALUES (13533,0,0,0,78,0,0,0,0,255,0,4096,4238,665,932,2001,2290,4022,0,0,0,8556,8556,8556,8556,12,12,12,12,'Armor Plate IV');
--- INSERT INTO `synth_recipes` VALUES (13534,0,0,255,79,27,0,0,0,0,0,4096,4238,650,652,654,711,0,0,0,0,17250,17261,17261,17261,1,1,1,1,'Matchlock Gun');
+ -- INSERT INTO `synth_recipes` VALUES (13533,0,0,0,78,0,0,0,0,34,0,4096,4238,665,932,2001,2290,4022,0,0,0,8556,8556,8556,8556,12,12,12,12,'Armor Plate IV'); -- JPWiki BUT ??AL--FFXIAH cites 34AL on 1st tier, same mats/qty for II/III/IV versions
```

Comment:
got a space in front of you dashes

Reply:
gah... never noticed that they fully aligned to left.  got too used to " -- " for comments at end of line.

important thing to remember.

edited out